app/src/main/res/layout/activity_passport_information.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:padding="@dimen/default_margin_medium">
```

Comment:
@Wadeewee Where did this value come from? It cannot be resolved on my end 😅

Reply:
Ops! 🙈 Wrong rebase. Thanks to spotted this 👍 @ryan-conway